pyaerocom/aeroval/coldatatojson_helpers.py

```diff
@@ -826,9 +826,9 @@ def _process_map_and_scat(
     scat_data = {}
     scat_dummy = [np.nan]
     for freq, cd in data.items():
-        use_dummy = True if cd is None else False
         for per in periods:
             for season in seasons:
+                use_dummy = True if cd is None else False
```

Comment:
``` python
        for per in periods:
            for season in seasons:
                use_dummy = cd is None
                ...
```

Reply:
I have to admit that rarely think of these short expressions because I don't find them very intuitive...
Done.